docs/concepts/storage/persistent-volumes.md

```diff
@@ -573,68 +573,51 @@ parameters:
 
 #### vSphere
 
-1. Create a persistent volume with a user specified disk format.
+   1. Create a StorageClass with a user specified disk format.
```

Comment:
The bullet formatting is now messed up. See https://deploy-preview-5465--kubernetes-io-master-staging.netlify.com/docs/concepts/storage/persistent-volumes/#vsphere

Do not indent the bullet numbers.